include/pybind11/detail/internals.h

```diff
@@ -182,6 +185,16 @@ struct internals {
 #    endif // PYBIND11_INTERNALS_VERSION > 4
     // Unused if PYBIND11_SIMPLE_GIL_MANAGEMENT is defined:
     PyInterpreterState *istate = nullptr;
+
+#    if PYBIND11_INTERNALS_VERSION > 4
+    // Note that we have to invoke strdup to allocate memory to ensure a unique address
```

Comment:
Sorry I missed this before: this still mentions strdup

Reply:
Oops, good catch!